### PR TITLE
Add proper ITS node

### PR DESCRIPTION
**Pull request by andr2000**

### CODE REVIEW

**Comment by andr2000** [comment]
N.B. This will stop the system from booting unless Spider's ITS is fully supported in Xen

File: meta-xt-domd-gen4/recipes-kernel/linux/linux-renesas/r8a779f0-spider-xen.dts [hunk 1]
```diff
@@ -66,6 +66,10 @@
 		 /* non-prefetchable memory */
 		 0x82000000 0 0x30000000 0 0x30000000 0 0x10000000>;
 	msi-map = <0x0 &gic_its 0x0 0x1000>;
+	interrupt-map = <0 0 0 1 &gic 0 0 GIC_SPI 416 IRQ_TYPE_LEVEL_HIGH
```
Comment (lorc) [inline meta-xt-domd-gen4/recipes-kernel/linux/linux-renesas/r8a779f0-spider-xen.dts:69]: Patch is named "Fix interrupt-map node", but it adds the node, not fixes it.

Comment (andr2000) [reply meta-xt-domd-gen4/recipes-kernel/linux/linux-renesas/r8a779f0-spider-xen.dts:69]: Well, it fixes the existing entry in the relevant dtsi file by replacing it here

Comment (andr2000) [reply meta-xt-domd-gen4/recipes-kernel/linux/linux-renesas/r8a779f0-spider-xen.dts:69]: `This is fixed by overriding the interrupt-map property from r8a779f0.dtsi.`
added to the commit message

**Comment by lorc** [comment]
`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Pull request merged**